### ISSUE TITLE
Ctrl+TAB, Ctrl+TAB key not working

### ISSUE THREAD

**Issue by isaacrlevin** [StageCoder/StageCoder-VS-Extension#3]
Trying to use StageCoder for the first time and it looks like some file switching hotkey in VS is blocking Type Snippet. I looked in hot keys but nothing came up, maybe you know

![stage](https://user-images.githubusercontent.com/8878502/235780609-a5f69bd1-7d8a-4753-94db-f0b40f2a8798.gif)

**Comment by EngstromJimmy** [comment]
I have noticed that in som cases you need to remap the hot key. Either you can map it to something else (if you want to keep ctrl+tab as is) or just map it to ctrl + tab tab

**Comment by EngstromJimmy** [comment]
The command you need to remap is StageCoder.Typecode

**Comment by isaacrlevin** [comment]
Yep I already remapped I was just calling to your attention that there is some native hotkey in VS 2022 Preview that will block you :)

**Comment by EngstromJimmy** [comment]
Got it, thanks :)

**Comment by praveen415** [comment]
<img width="326" alt="Image" src="https://github.com/user-attachments/assets/8e8d67ea-15b5-46f7-a3b1-305da9c30a4d" />

Highlighted the code and right clicked to create snippet , but have an error . please help me .

**Comment by EngstromJimmy** [comment]
@praveen415 could you also send me the file the error message is refering to?

**Comment by praveen415** [comment]
[ActivityLog.txt](https://github.com/user-attachments/files/20489179/ActivityLog.txt)

Thanks for the prompt reply , here is the Activity log file , please help to fix this. It will be very helpful to me . I'm using Visual Studio 2022 community edition . Thanks in advance

**Comment by praveen415** [comment]
Upgraded my Visual Studio version and it worked fine , Thanks alot

**Comment by EngstromJimmy** [comment]
Awesome!
